R/render.r

```diff
@@ -482,7 +596,17 @@ check_bootswatch_theme <- function(bootswatch_theme, bs_version, pkg) {
 }
 
 pkgdown_bslib_defaults <- function() {
-  list(`navbar-nav-link-padding-x` = "1rem")
+  list(
+    `navbar-nav-link-padding-x` = "1rem",
+    `primary` = "#0054AD",
+    `secondary` = "#e9ecef",
+    `navbar-bg` = "#f8f9fa",
+    `border-width` = "1px",
+    `code-bg` = "#f8f8f8",
+    `code-color` = "#333",
+    `fu-color` = "#4758AB",
+    `border-radius` = "1rem"
+  )
 }
```

Comment:
I prefer putting these variable definitons in the Sass code (with `!default` flags). That way it's easier to reason about the Sass code (plus, the Sass file can then we compiled in a standalone fashion)

Reply:
@cpsievert Good points but I am not sure how to do that. The code below does not work because I'm using `bs_add_rules` which seems to ignore variables. How should I best combine the sass code with the bslib theme?

``` r
sass_code <- "$primary: pink !default;"
bs_theme <- bslib::bs_theme(version = 4)
new_css <- sass::sass_partial(sass_code, bs_theme)
bs_theme <- bslib::bs_add_rules(bs_theme, new_css)
bslib::bs_get_variables(bs_theme, "$primary")
#>   primary 
#> "#007bff"
```

<sup>Created on 2021-07-13 by the [reprex package](https://reprex.tidyverse.org) (v2.0.0)</sup>

Reply:
Exploring in https://github.com/maelle/bslib-stuff/pull/1#issuecomment-879722320